src/components/ButtonPanel.js

```diff
@@ -1,11 +1,11 @@
+/* eslint-disable react/jsx-filename-extension */
 import React from 'react';
 import Button from './Button';
 
-
-class ButtonPanel extends React.Component{
-  render(){
-    return(
-        <Button onClick={this.onClick}/>
+class ButtonPanel extends React.Component {
+  render() {
+    return (
+      <Button onClick={this.onClick} />
     );
   }
```

Comment:
The `ButtonPanel` should render the `Button` component here using a group of `name` props

![image](https://user-images.githubusercontent.com/36057474/72732580-4a8b5580-3b96-11ea-8afd-86964ac43ec1.png)

Reply:
I don't really understand what you want here though. My understanding is that ButtonPanel will render Buttons. Buttons are the symbols and number isn't it? Should there not be at least a bit of freedom on expressing what I read from the instructions? or there is another way to it?

Reply:
I still do not understand what you are saying here. When I mentioned "freedom" I am referring to the way I understand the requirements. Now should we all do things the same way? or is this an attempt to kill the creativity of student? is it mandatory that I go this route? Now I think there is an issue with the way the requirement was crafted by Microverse in my own humble opinion and it ought to be corrected at that time noot during code review. Moving forward,  ButtonPanel to me is a container for buttons and it is my real-world interpretation of the relationship between the Buttonpanel and Button. Button is a single calculator button and the Panel holds the Button. I don't know if you get my drift here? That is the crux of my complaint. and that is what I implemented in this milestone.

Reply:
Hi @tundeiness 

Sorry my explanation wasn't clear enough.

Just like you said Button should be a single re-usable button of the calculator while BottunPanel holds all the Button components required to complete a full button panel